mage/operator/lib.go

```diff
@@ -25,9 +25,11 @@ import (
 const (
 	rootBuildDir               = "./build"
 	sdkBuildDir                = "operator/build"
+	diagramsDir                = "./docs/developer/diagrams"
 	operatorSdkImage           = "operator-sdk-binary"
 	testSdkImage               = "operator-sdk-binary-tester"
 	genClientImage             = "operator-gen-client"
+	mermaidJsImage             = "operator-mermaid-js"
```

Comment:
Prefixing with "operator-" to prevent collision with any _actual_ mermaid-js image a user might have. As far as I know, there is not currently an official one.